.github/workflows/ci.yaml

```diff
@@ -60,6 +60,74 @@ jobs:
           name: gradle_spotbugs_reports
           path: /tmp/gradle_spotbugs_reports
 
+  docker:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
```

Comment:
My understanding is that `gradle/gradle-build-action@v2` action will [already cache gradle dependency](https://github.com/gradle/gradle-build-action#caching) making the usage of `actions/cache@v2` redundant. What do you think ?

Reply:
Removed the Cache part in the latest commit. Thanks.